File: LP1/php/DPR.php
<?php


class DPR
{
    private $id = -1;
    private $nama = "";
    private $bidang = "";
    private $partai = "";
    private $foto = "";

    public function __construct($id, $nama, $bidang, $partai, $foto)
    {
        $this->id - $id;
        $this->nama = $nama;
        $this->bidang = $bidang;
        $this->partai = $partai;
        $this->foto = $foto;
    }


    // setter getter
    // id
    public function getID(): int
    {
        return $this->id;
    }
    public function setID($id)
    {
        $this->id = $id;
    }
    // nama
    public function getNama(): string
    {
        return $this->nama;
    }
    public function setNama($nama)
    {
        $this->nama = $nama;
    }
    // bidang
    public function getBidang(): string
    {
        return $this->bidang;
    }
    public function setBidang($bidang)
    {
        $this->bidang = $bidang;
    }
    // partai
    public function getPartai(): string
    {
        return $this->partai;
    }
    public function setPartai($partai)
    {
        $this->partai = $partai;
    }
    // foto
    public function getImg(): string
    {
        return $this->foto;
    }

    public function setImg($foto)
    {
        $this->foto = $foto;
    }
}

File: LP1/python/DPR.py
class DPR:
    __ID = -1
    __name = ""
    __partai = ""
    __bidang = ""

    # Python only use 1 constructor
    def __init__(self, id, name="", bidang="", partai=""):
        self.__ID = id
        self.__name = name
        self.__bidang = bidang
        self.__partai = partai

    # Getter Setter

    # ID
    def getID(self):
        return self.__ID

    def setID(self, ID):
        self.__ID = ID

    # NAME
    def getName(self):
        return self.__name

    def setName(self, name):
        self.__name = name

    # Bidang
    def getBidang(self):
        return self.__bidang

    def setBidang(self, bidang):
        self.__bidang = bidang

    # Partai
    def getPartai(self):
        return self.__partai

    def setPartai(self, partai):
        self.__partai = partai
